firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigComponent.java

```diff
@@ -262,7 +262,8 @@ ConfigFetchHttpClient getFrcBackendApiClient(
         apiKey,
         namespace,
         /* connectTimeoutInSeconds= */ metadataClient.getFetchTimeoutInSeconds(),
-        /* readTimeoutInSeconds= */ metadataClient.getFetchTimeoutInSeconds());
+        /* readTimeoutInSeconds= */ metadataClient.getFetchTimeoutInSeconds(),
```

Comment:
you could merge https://github.com/firebase/firebase-android-sdk/pull/6440 and have the naming change reflected here

Reply:
Done